grails-app/domain/org/pih/warehouse/shipping/ShipmentItem.groovy

```diff
@@ -224,6 +224,11 @@ class ShipmentItem implements Comparable, Serializable {
         return orderItem ? orderItem.quantityPerUom : 1
     }
 
+    String getUnitOfMeasure() {
+        OrderItem orderItem = OrderItem.get(this.orderItemId)
```

Comment:
Looking at how the `orderItemId` transient is implemented, I don't see a point in fetching the order item here. I'd rather just go with `orderItems[0]`